Readme.md

```diff
@@ -162,8 +162,8 @@ This event is triggered if any javscript exception is thrown on the page. But th
 ##### .on('page-log', errorMessage, errorStack)
 This event is triggered if `console.log` is used on the page. But this event is not triggered if the injected javascript code (e.g. via `.evaluate()`) is using `console.log`.
 
-#### .screenshot(path[, clip])
-Saves a screenshot of the current page to the specified `path`. Useful for debugging. The output is always a `png`. You can optionally provide a clip rect as [documented here](https://github.com/atom/electron/blob/master/docs/api/browser-window.md#wincapturepagerect-callback).
+#### .screenshot([path, clip])
```

Comment:
[Here's](https://nodejs.org/api/fs.html#fs_fs_watch_filename_options_listener) some inspiration from the node API. They use

    fs.watch(filename[, options][, listener])

for a function that can take `filename`, or `filename, options`, or `filename, listener`, or all three. So how about

    .screenshot([path][, clip])

I can also add a two or three line example of how the path, the clip, or both can be omitted.